CHANGELOG.md

```diff
@@ -1,167 +1,169 @@
 # Change Log
 All notable changes to the "ms-vscode.azure-account" extension will be documented in this file.
 
+## 0.9.9 - 2021-07-09
```

Comment:
Should we conform this even more to how we normally release?

For example, the headers `## Added/## Fixed`.  Linking the closed issues of the milestone.  Anything else I might be missing.

Reply:
I'll keep this in mind for the next changelog change I make here. But closing this PR as discussed offline